www/src/utils/getParagonComponentsTypes.js

```diff
@@ -11,9 +11,6 @@ const getParagonComponentsTypes = (components) => {
       case typeof component === 'string' || typeof component === 'number':
         componentType = 'Text';
         break;
-      case component.name?.startsWith('use'):
```

Comment:
can you check it this way `case componentName.startsWith('use')`? It seems to work for me locally with production and dev builds